### PR TITLE
Chore: add codeowners

### PR DESCRIPTION
**Pull request by jdkandersson**

### CODE REVIEW

**Review by arturo-seijas — DISMISSED** [review]
LGTM

**Comment by jdkandersson** [comment]
The integration test is likely failing due to the current production problem with Jenkins, will wait for that problem to be fixed before merging

**Review by merkata — DISMISSED** [review]
Nice addition, approving.

File: tox.ini [hunk 1]
```diff
@@ -76,4 +76,3 @@ commands =
     pylint {[vars]all_path}
     isort --check-only --diff {[vars]all_path}
     black --check --diff {[vars]all_path}
-    mypy {[vars]all_path}
```
Comment (merkata) [inline tox.ini:79]: Is that removal intentional and part of the PR?

Comment (yhaliaw) [reply tox.ini:79]: Yes, the mypy check are moved to a different story now.

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by merkata — APPROVED** [review]
Good, approving.
